privacyidea/lib/tokens/smstoken.py

```diff
@@ -349,10 +349,10 @@ def create_challenge(self, transactionid=None, options=None):
                 db_challenge.save()
                 transactionid = transactionid or db_challenge.transaction_id
             except Exception as e:
-                info = ("The PIN was correct, but the "
-                        "SMS could not be sent: %r" % e)
-                log.warning(info)
-                log.debug("{0!s}".format(traceback.format_exc()))
+                info = _("The PIN was correct, but the "
+                         "SMS could not be sent.")
+                log.warning(info + " ({0!s})".format(e))
+                log.debug(u"{0!s}".format(traceback.format_exc()))
```

Comment:
the `u` is not needed here, there shouldn't be any unicode issues in the stack-trace.